C-Plus-Plus/ds/Circular_Queue.cpp

```diff
@@ -0,0 +1,261 @@
+/*
+This is a code of circular queue data structure. It is a linear data structure, which obeys the principal
+ of First in First Out(FIFO). Circular queue data structure is a updated version of linear queue data structure,
+ which have some advantages over linear queue. The following code shows the implementation of circular queue data structure,
+ in a very clean and understandable way.
+*/
+
+#include<iostream>
+#include<stdlib.h>
+
+// Defining max size of the queue as 5
+
+#define max 5
+
+using namespace std;
+
+// Writing a class where all the basic functions of the circular queue presides
+
+class CircularQueue{
+    /*
+    Defining all the variables as public variables.
```

Comment:
Inconsistent indentation.

Reply:
Sir I corrected the indentation and suggested changes. Please check it sir.